generator/internal/descriptor_utils.cc

```diff
@@ -396,6 +396,17 @@ ParameterCommentSubstitution substitutions[] = {
     {R"""(`{cluster} = '-'`)""", R"""(``{cluster} = '-'``)"""},
     {R"""(`projects/<Project ID or '-'>`)""",
      R"""(``projects/<Project ID or '-'>``)"""},
+
+    // Some comments include multiple newlines in a row. We need to preserve
+    // these because they are paragraph separators. When used in `@param`
+    // commands we need to represent them as `@n` or they do more than separate
+    // the paragraph. The would terminate the `@param` comment.
```

Comment:
"we need to represent them as `@n` or they would terminate the `@param` comment."